### PR TITLE
Show all matching transcripts

### PR DESCRIPTION
**Pull request by foglabs**
Show them

### CODE REVIEW

File: gulp/js/router.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ app.routers.DefaultRouter = Backbone.Router.extend({
     "transcripts/:id?*queryString": "transcriptEdit",
     "page/:id":                     "pageShow",
     "dashboard":                    "dashboard",
-    "search":                       "search",
-    "search?*queryString":          "search"
+    // "search":                       "search",
```
Comment (jasoncorum) [inline gulp/js/router.js]: On this commented out code and the following is there a reason to keep it here for now?

File: lib/tasks/aapb.rake [hunk 2]
```diff
@@ -11,8 +11,11 @@ namespace :aapb do
     raise "Not a valid ids_file_path #{args[:ids_file_path]}" unless File.exist?(args[:ids_file_path])
     raise "Not a valid project_key" unless Dir.exist?(Rails.root.join('project', args[:project_key]))
 
-    ids = build_ids_array(args[:ids_file_path])
-    AAPB::TranscriptSeedsJob.new(ids,args[:project_key]).run!
+    ids_file_path = '/Users/henry_neels/gbh/transcript-editor/ids'
```
Comment (jasoncorum) [inline lib/tasks/aapb.rake]: This should probably be removed.

File: public/assets/js/default.js [hunk 3]
```diff
@@ -732,8 +732,8 @@ app.routers.DefaultRouter = Backbone.Router.extend({
     "transcripts/:id?*queryString": "transcriptEdit",
     "page/:id":                     "pageShow",
     "dashboard":                    "dashboard",
-    "search":                       "search",
-    "search?*queryString":          "search"
+    // "search":                       "search",
```
Comment (jasoncorum) [inline public/assets/js/default.js]: Same question as above commented code.

File: lib/tasks/aapb.rake (hunk 2 above)
Comment (foglabs) [reply lib/tasks/aapb.rake]: whoops!

File: gulp/js/router.js (hunk 1 above)
Comment (foglabs) [reply gulp/js/router.js]: I'll delete it

**Pull request merged**